### PR TITLE
Feature/fair 428

### PR DESCRIPTION
**Pull request by ho-henry-senior**
Updated crown image to new version:
- favicon set
- svg in logged in and not logged in page

image

### CODE REVIEW

**Review by ame-test — COMMENTED** [review]
The below question is that `mask-icon` lead to non existent image being shown on Safari.
* I appreciate the desire to merge, but it should be checked.
* I look to approve soon though.

File: src/common/assets/images/govuk-mask-icon.svg [hunk 1]
```diff
@@ -1,7 +0,0 @@
...
....1 18.4 69.4 18 69.7 17.7L69.7 17.7ZM66 92.8C82.9 92.8 98.8 93.9 113.1 96 117.1 79.1 122 69.3 127.1 62.5L117.5 59.1C118.5 64 118.6 66.3 117.5 69.3 116 67.9 114.5 65 113.3 60.6L108.6 76C111.4 74 113.6 72.8 116.1 72.7 111.7 82.1 106.1 84.6 102.5 83.9 98.2 83.1 96.2 79.3 96.9 76 97.9 71.3 102.6 70.1 104.9 75.5 109.2 66.8 101.9 64.1 97.3 66.7 104.4 59.5 105.2 53.2 99.4 45.6 91.4 51.7 91.3 57.9 94.9 66.4 90.2 61 82.8 63.9 85.4 72.6 88.8 67.4 93.3 70.6 92.6 75.7 92 80 86.2 83.5 79.1 82.9 68.8 82 68.2 74.9 67.9 69.1 70.4 68.6 75 70.9 78.9 76.4L80.2 60C76.1 64.4 72.2 65.3 67.9 65.4 69.3 61 75.9 53.8 75.9 53.8L66.4 53.8 66.4 53.8 66.4 53.8 55.5 53.8C55.5 53.8 61.9 61 63.4 65.4 59.2 65.3 55.4 64.4 51.1 60L52.5 76.4C56.4 70.9 61 68.7 63.4 69.1 63.1 74.9 62.5 81.9 52.3 82.9 45.1 83.5 39.4 80 38.8 75.7 38.1 70.7 42.6 67.4 45.9 72.6 48.6 63.9 41.3 61 36.5 66.4 40.2 57.9 40.1 51.7 31.9 45.6 26.1 53.2 26.9 59.5 34.1 66.7 29.4 64.1 22.2 66.8 26.4 75.5 28.7 70 33.5 71.3 34.5 76 35.2 79.3 33.2 83.1 28.8 83.9 25.3 84.6 19.8 82.1 15.3 72.7 17.8 72.8 20 74 22.8 76L18.1 60.6C16.9 65 15.4 67.8 13.8 69.3 12.7 66.3 12.9 64 13.8 59.1L4.3 62.5C9.3 69.4 14.2 79.2 18.3 96 33.1 93.9 49.1 92.8 66 92.8L66 92.8Z"/>
```
[the remark this answers is not in the document]
Comment (ho-henry-senior) [reply src/common/assets/images/govuk-mask-icon.svg:4]: image
mask-icon added and tested in Safari

File: src/common/templates/includes/header-pre-user.njk [hunk 2]
```diff
@@ -26,8 +25,7 @@
       <nav>
         <ul id="navigation" class="govuk-header__navigation" aria-label="Top Level Navigation">
           {{ nav_link('upt-help', '/help', __('nav_bar_upt-help')) }}
-        </ul>
-      </nav>
-    </div>
+      </ul>
+    </nav>
   </div>
```
Comment (ame-test) [inline src/common/templates/includes/header-pre-user.njk:30]: Also missing closing div is this an issue or not?

File: src/common/assets/images/govuk-mask-icon.svg (hunk 1 above)
[the remark this answers is not in the document]
Comment (ame-test) [reply src/common/assets/images/govuk-mask-icon.svg:4]: Done.

**Review by ame-test — APPROVED** [review]
Html bit should be fixed, but the mask-icon bit is done, so I am happy enough.

**Pull request merged**